recipes/speclite/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "speclite" %}
+{% set version = "0.7" %}
+{% set sha256 = "2753137e2f3dc0b94d6c3f98c24d674c1a7145410d731cbeb13e0e7f1f7f5ba2" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  sha256: {{sha256}}
+  
+build:
+  number: 0
+  noarch: python
+
+requirements:
+
+  build:
+    - python
+    - setuptools
+    - pip
+    - wget
+    - numpy
+    - scipy
+    - astropy
+    - matplotlib
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - astropy
+    - matplotlib
+
+test:
+
+  imports:
+    - speclite
+    
+  commands:
+    # Execute the full test suite
+    
+    - cd ${PREFIX}/lib/python*/site-packages/speclite/tests ; py.test -vv
+  
+  requires:
+    - pytest
+    - pyyaml
+
+about:
+  home: http://speclite.readthedocs.io/en/latest/overview.html
+  license: BSD
+  license_family: BSD
+  summary: 'This package provides a set of lightweight utilities for working with spectroscopic data in astronomy. Based on the astropy affiliated package template.'
+  license_file: '{{ environ["SRC_DIR"] }}/licenses/LICENSE.rst'
```

Comment:
Please raise an issue upstream if the `license_file` is missing from the source distribution and add the link as a comment in the recipe.

Reply:
it is not missing in the source distribution, but it is in a subdirectory.